src/core/algorithms/gwo.cpp

```diff
@@ -31,31 +31,41 @@ void gwo::reset()
 void gwo::update_elite()
 {
   float size = get_solutions()->size();
-  for(int i = 0; i < size; i++){
+
+  for(int i = 0; i < size; i++)
+  {
     solution* current = get_solutions()->get(i);
-    if (current->is_better_than(_alpha,is_maximization())) {
+    if (current->is_better_than(_alpha,is_maximization()))
```

Comment:
After the colon `,` you should leave a space. The same issue happens in several lines.

Reply:
This line: ` solution* current = get_solutions()->get(i);` is a variable declaration, hence, you should leave an empty line after this block